env/test

```diff
@@ -1,7 +1,8 @@
 # Add Web3 url
-NODE_URL=/*INSERT_HERE*/
+NODE_URL=https://ropsten.infura.io/v3/7bfab7398ae84af3b1b70c955cfd9491
 
 # Ethereum private key prefixed with 0x
-PRIVATE_ETH_KEY=0x/*INSERT_HERE*/
-PUBLIC_ETH_ADDRESS=0x/*INSERT_HERE*/
-PRIVATE_STARK_KEY=/*INSERT_HERE*/
+PRIVATE_ETH_KEY=0x49e4d1e2aa7d026188251392dd2d335c176d846d8a894a8092c835f3b345e2ad
+PUBLIC_ETH_ADDRESS=0x49e4d1e2aa7d026188251392dd2d335c176d846d8a894a8092c835f3b345e2ad
+PRIVATE_STARK_KEY=0x49e4d1e2aa7d026188251392dd2d335c176d846d8a894a8092c835f3b345e2ad
+INFURA_PROJECT_ID=af1d9fbeceb44e76be1bb93738f00d19
```

Comment:
How about using sops to encrypt these?

Reply:
I did that way to make it easier to run the tests for now, since we don't have the full routine yet I didn't spend time worrying about that.
I can definitely work on it and let it be all setup from now on